pandera/api/polars/types.py

```diff
@@ -20,7 +20,8 @@ class CheckResult(NamedTuple):
 
 
 PolarsCheckObjects = Union[pl.LazyFrame, pl.DataFrame]
-
+PolarsFrame = TypeVar("PolarsFrame", pl.LazyFrame, pl.DataFrame)
```

Comment:
I wasn't sure if these should go here or somewhere else, since some packages treat api packages a public facing, but I saw PolarsCheckObjects and PolarsData are here and they're not exposed externally.